### ISSUE TITLE
[Docs - New Feature]: carousel component navigable without scrolling

### ISSUE THREAD

**Issue by filiptronicek** [gitpod-io/website#1022]
### Describe your feature

In blog posts like "VS Code in the browser for everyone", where there is a carousel (this one being composed of quotes), it would make a great UX and a11y to add more ways to move the carousel around.

One such mechanism would be to have the next item clickable (with a pointer cursor), to not force the user to scroll. 

Another one which comes to mind would be to make each quote focusable, so one could navigate the carousel all with the keyboard. 

### How to test

It should be focusable with <kbd>Tab</kbd> and trigger-able with <kbd>Enter</kbd> or <kbd>Space</kbd>.
